pynautobot/core/api.py

```diff
@@ -75,6 +77,7 @@ def __init__(
         token=None,
         threading=False,
         api_version=None,
+        api_depth=os.getenv("PYNAUTOBOT_API_DEPTH"),
```

Comment:
Should we add a default depth of 1 here? Or did we decide against that?

```suggestion
        api_depth=int(os.getenv("PYNAUTOBOT_API_DEPTH", "1")),
```

I think we will want it as an integer as well? I believe?

Reply:
We didn't clearly decide, but I think it's reasonable to set it for `1`.